packages/legend-query/src/stores/LegendQueryStore.ts

```diff
@@ -721,6 +723,45 @@ export class LegendQueryStore {
     }
   }
 
+  *fetchProjectEntities(
```

Comment:
I have mixed-feelings about this. It's decent, but it tears away the analytics we have using `stopWatch`. I believe this might screw up `getRecord(...)` call later when we build the report for graph building. All in all, I think we should revert this.